yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -219,8 +219,8 @@ class TakeOperator : public FlowableOperator<T, T> {
         FlowableOperator<T, T>::Subscription::subscriber_->onNext(
             std::move(value));
         if (limit_ == 0) {
+          FlowableOperator<T, T>::Subscription::subscriber_->onComplete();
```

Comment:
I believe we should also update onComplete() method in the FlowableOperator to check if the Subscription is null. At least we might assert in the Debug build, if the subscription is null and if we call onComplete(). 

Furthermore, in FlowableOperator, we call Release multiple times. We might assert in the Release function also. Because it is reducing the reference count more than once per call.